quinn-proto/src/connection/mod.rs

```diff
@@ -4290,10 +4290,16 @@ impl Connection {
                         path.data.challenges_sent.clear();
                         path.data.send_new_challenge = false;
                         path.data.validated = true;
-                        path.data.rtt.update(
-                            Duration::ZERO,
-                            now.saturating_duration_since(challenge_sent),
-                        );
+                        if path.data.total_recvd == 0 {
```

Comment:
I'm not sure this condition is correct, since having an RTT sample has a series of conditions that might not occurr, even if we have received data